src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -199,12 +199,13 @@ private async Task PerformV2RestoreAsync(string packagesConfigFilePath, string i
                 cacheContext.NoCache = NoCache;
                 cacheContext.DirectDownload = DirectDownload;
 
-                var downloadContext = new PackageDownloadContext(cacheContext, installPath, DirectDownload);
+                var downloadContext = new PackageDownloadContext(cacheContext, Settings, installPath, DirectDownload);
 
                 var result = await PackageRestoreManager.RestoreMissingPackagesAsync(
                     packageRestoreContext,
                     new ConsoleProjectContext(Console),
-                    downloadContext);
+                    downloadContext,
+                    Settings);
```

Comment:
Do you need to pass `Settings`?  Can't `RestoreMissingPackagesAsync(...)` get it from `downloadContext.Settings`?